schedule/migration/migration_offline_media_x86_64.yaml

```diff
@@ -26,10 +26,10 @@ schedule:
   - installation/scc_registration
   - installation/addon_products_sle
   - installation/resolve_dependency_issues
-  - installation/installation_overview
-  - installation/disable_grub_timeout
-  - installation/start_install
-  - installation/await_install
+  - installation/bootloader_settings/disable_boot_menu_timeout
```

Comment:
and modules like registration or welcome? looks like there are more modules that could be enabled. If it is not possible, please file ticket to covert to libyui-rest-api.

Reply:
then there is resolve_dependency_issues, we can probably get rid of it, but I was not sure, as ths is migraton, it could actually happen to be useful sometimes (?)